src/LibLog/LibLog.cs

```diff
@@ -561,6 +617,7 @@ internal static ILogProvider ResolveLogProvider()
             return null;
         }
 
+#if !LIBLOG_PROVIDERS_ONLY
```

Comment:
Yep, the final API has no `ILog` because it doesn't need one. This would have no use.